### PR TITLE
net-analyzer/hexinject: new package

### PR DESCRIPTION
**Pull request by kolodziej**
Package-Manager: portage-2.3.2

### CODE REVIEW

File: net-analyzer/hexinject/hexinject-1.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Hexadecimal and raw packet injector and sniffer. Can be easily
```
Comment (SoapGentoo) [inline net-analyzer/hexinject/hexinject-1.5.ebuild]: `DESCRIPTION` is way too long. If you want a long description, put it in `metadata.xml` (which you already have). This shouldn't be more than 80 chars.

File: net-analyzer/hexinject/hexinject-1.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+SRC_URI="http://downloads.sourceforge.net/project/${PN}/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="tools -experimental"
+
+DEPEND="net-libs/libpcap"
+RDEPEND="${DEPEND}
+		experimental? ( dev-lang/tcl )"
```
Comment (SoapGentoo) [inline net-analyzer/hexinject/hexinject-1.5.ebuild]: too much indentation (one tab)

File: net-analyzer/hexinject/hexinject-1.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+DESCRIPTION="Hexadecimal and raw packet injector and sniffer. Can be easily
+combined with other tools to provide a powerful cmdline framework for raw
+network access."
+HOMEPAGE="http://hexinject.sourceforge.net/"
+SRC_URI="http://downloads.sourceforge.net/project/${PN}/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="tools -experimental"
```
Comment (mgorny) [inline net-analyzer/hexinject/hexinject-1.5.ebuild]: Don't use `-stuff` in IUSE. Just specifying it makes it off by default.

File: net-analyzer/hexinject/hexinject-1.5.ebuild (hunk 1 above)
Comment (mgorny) [inline net-analyzer/hexinject/hexinject-1.5.ebuild]: Strictly speaking, I think the limit of 80 chars includes `DESCRIPTION=""` part. And don't end description with a dot if it's not a proper sentence.

File: net-analyzer/hexinject/metadata.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>kacper@kolodziej.in</email>
+		<name>Kacper Kołodziej</name>
+	</maintainer>
+	<longdescription lang="en">
```
Comment (mgorny) [inline net-analyzer/hexinject/metadata.xml]: Add proxy-maint here.

Comment (kolodziej) [inline net-analyzer/hexinject/metadata.xml]: Add or replace me with proxy-maint?

**Comment by doughdemon** [comment]
- The upstream Makefile harcodes the compiler, the CFLAGS and LDFLAGS.
- Some patching is necessary to be able to compile with gcc-5: `sed -i 's/inline/static inline/' prettypacket.h`

**Comment by SoapGentoo** [comment]
@doughdemon either that or `append-cflags -std=gnu89`

File: net-analyzer/hexinject/metadata.xml (hunk 4 above)
Comment (mgorny) [inline net-analyzer/hexinject/metadata.xml]: Add. All bugs will still be assigned to you, and proxy-maint will be CC-ed to help you and take action when you go MIA ;-).

**Comment by SoapGentoo** [comment]
Fixed it myself

**Pull request closed without merging**